notebooks/search/05-query-rules.ipynb

```diff
@@ -0,0 +1,437 @@
+{
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "id": "87773ce7",
+      "metadata": {
+        "id": "87773ce7"
+      },
+      "source": [
+        "# Search using query rules\n",
+        "\n",
+        "<a target=\"_blank\" href=\"https://colab.research.google.com/github/kderusso/elasticsearch-labs/blob/kderusso/query-rules-notebook/notebooks/search/05-query-rules.ipynb\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>\n",
+        "\n",
+        "This interactive notebook will introduce you to how use query rules, using the official [Elasticsearch Python client](https://www.elastic.co/guide/en/elasticsearch/client/python-api/current/connecting.html).\n",
+        "You'll store query rules in Elasticsearch using the [query rules API](https://www.elastic.co/guide/en/elasticsearch/reference/current/query-rules-apis.html) and query them using [rule_query](https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-rule-query.html)."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "id": "a32202e2",
+      "metadata": {
+        "id": "a32202e2"
+      },
+      "source": [
+        "## Create Elastic Cloud deployment\n",
+        "\n",
+        "If you don't have an Elastic Cloud deployment, sign up [here](https://cloud.elastic.co/registration?fromURI=%2Fhome) for a free trial.\n",
+        "\n",
+        "- Go to the [Create deployment](https://cloud.elastic.co/deployments/create) page\n",
+        "   - Select **Create deployment**"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "id": "52a6a607",
+      "metadata": {
+        "id": "52a6a607"
+      },
+      "source": [
+        "## Install packages and import modules\n",
+        "\n",
+        "To get started, we'll need to connect to our Elastic deployment using the Python client.\n",
+        "Because we're using an Elastic Cloud deployment, we'll use the **Cloud ID** to identify our deployment.\n",
+        "\n",
+        "First we need to install the `elasticsearch` Python client."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "id": "ffc5fa6f",
+      "metadata": {
+        "id": "ffc5fa6f",
+        "scrolled": false
+      },
+      "outputs": [],
+      "source": [
+        "!pip install -qU elasticsearch sentence-transformers==2.2.2"
```

Comment:
nitpick: dont need sentence transformers here

Reply:
Thanks for the catch! You caught me on copy-pasta 😂